Sample.java

```diff
@@ -3,5 +3,18 @@
 // Did this code successfully run on Leetcode :
 // Any problem you faced while coding this :
 
+MyHashSet Implementation
+Space and time complexity : O(n)
+Runtime: 10 ms, faster than 88.11% of Java online submissions for Design HashSet.
+Memory Usage: 45.1 MB, less than 95.77% of Java online submissions for Design HashSet.
```

Comment:
No need to add Runtime and Memory stats from Leetcode, Please only add the asymptotic Time & Space Complexity. Please copy over this template to the beginning of each file and modify it there.